Sources/RPListener.swift

```diff
@@ -66,13 +67,20 @@ public class RPListener: NSObject, XCTestObservation {
       buildVersion: buildVersion,
       testType: testType.rawValue,
       testPriority: testPriority.rawValue
-      )
+    )
   }
-    
+
   public func testBundleWillStart(_ testBundle: Bundle) {
     self.configuration = readConfiguration(from: testBundle)
-    
-    guard configuration.shouldSendReport else {
+
+    //Determine whether to send data to the Report Portal. Data can be sent if tests are run
+    //from CircleCI or the PushTestDataToReportPortal parameter is set to YES
+    let circleCIRun = (ProcessInfo.processInfo.environment["CIRCLECI"] ?? "false") == "true"
+    if  circleCIRun || configuration.shouldSendReport {
+        publishData = true
+    }
```

Comment:
`nit`: The `if` statement can be removed and replaced with
```suggestion
    publishData = circleCIRun || configuration.shouldSendReport
```